### DISCUSSION TITLE
自定义视图组件内容变更如何与代码同步

### DISCUSSION THREAD

**Discussion by lusida** [opensumi/core#1919] [Q&A]
![企业微信截图_16675263702181](https://user-images.githubusercontent.com/43487524/199867429-2a817583-3fda-4d7d-9a7c-66c331e6c6aa.png)

自定义View将代码内容提供可视化操作，那自定义视图内容变更重新生成代码如何同步Code的内容？

**Comment by erha19** [comment]
@lusida 你可以通过 `IFileServiceClient` 去监听文件变化及修改相应文件内容，这个需要你自行实现相关能力。

**Reply by lusida** [reply]
只是内容变更，并未保存到文件，应该监听不到文件变化
